schema/check_test_output.py

```diff
@@ -77,9 +77,21 @@ def main(args):
     schema_data_results = []
     schema_count = 0
 
-    all_results = validator.validate_test_output_with_schema()
+    all_results, test_validation_plans = validator.validate_test_output_with_schema()
     logging.info('  %d results for test output', len(all_results))
 
+    # Check if any files in the expected list were not validated.
+    test_paths = []
```

Comment:
Make this a set if its main purpose is to check for `contains` and order doesn't matter.

This is true of lists vs. sets in general, and this SO answer about [Python Sets vs Lists](https://stackoverflow.com/questions/2831212/python-sets-vs-lists) confirms that Python is no different.

Reply:
Sure